apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts

```diff
@@ -68,12 +75,19 @@ export class VoMembersComponent implements OnInit {
     this.membersService.findCompleteRichMembers(this.vo.id, this.searchString, this.attrNames).subscribe(
       members => {
         this.members = members;
+        this.filterBasedOnStatus();
         this.loading = false;
       },
       () => this.loading = false
     );
   }
 
+  filterBasedOnStatus() {
```

Comment:
Remove this method completely. It doesn't make sense to filter results that should be already filtered on the backend.